tests/utils.py

```diff
@@ -0,0 +1,51 @@
+import string
+import time
+from random import choice, randint
+
+from matrix_client.errors import MatrixRequestError
+
+from raiden.network.transport.matrix.client import GMatrixHttpApi
+from raiden.tests.mocked.test_matrix_transport import \
+    create_new_users_for_address
+
+
+class TestGMatrixHttpApi(GMatrixHttpApi):
+    def __init__(self, server_name, user_presence=dict(), activity_change=dict()):
+        super(TestGMatrixHttpApi, self).__init__(base_url=server_name)
+        self.server_name = server_name
+        self.user_presence = user_presence
+        self.activity_change = activity_change
+
+    def get_room_id(self, room_alias) -> str:
+        letters = string.ascii_lowercase
+        random_room_id = "".join(choice(letters) for i in range(10))
+        return f"!{random_room_id}:{self.server_name}"
+
+    def get_presence(self, user_id):
+        if user_id in self.activity_change:
+            return {"presence": "online", "last_active_ago": self.activity_change[user_id] * 1000}
+        if user_id in self.user_presence:
+            return {"presence": "offline", "last_active_ago": self.user_presence[user_id] * 1000}
+
+        raise MatrixRequestError()
+
+    def kick_user(self, room_id, user_id):
+        return None
+
+    def get_joined_members(self, room_id):
+        return self.user_presence
+
+    def get_room_members(self, room_id):
+        return {"chunk": []}
+
+
+def create_user_activity_dict(size, lower_bound, upper_bound):
```

Comment:
The bounds are ages in seconds, not unix timestamps as the output. Renaming the variables or adding a docstring would be helpful when reading.